src/helpers/loadFont.ts

```diff
@@ -5,7 +5,7 @@ export default (url: string) => new Promise((resolve) => {
   }
 
   if (document.head.querySelector(`link[href="${url}"]`)) {
-    resolve();
+    resolve({});
```

Comment:
Ok, I've looked one more time.
It seems like we need to declare that this promise is returning `void` and then fix a little bit the overwrite for `onload`
```ts
export default (url: string) => new Promise<void>((resolve) => {
  if (typeof(document) === 'undefined' || !document.head) {
    throw new Error('fonts can only be loaded in the browser');
  }

  if (document.head.querySelector(`link[href="${url}"]`)) {
    resolve();
    return;
  }

  const link = document.createElement('link');
  link.setAttribute('rel', 'stylesheet');
  link.setAttribute('href', url);
  link.onload = () => resolve();
  document.head.appendChild(link);
});
```